src/types.ts

```diff
@@ -192,30 +192,37 @@ type BotQuickStrategyForm = {
         | 'loss_threshold_warning_popup'
         | 'learn_more_expansion'
         | 'learn_more_collapse'
-        | 'edit_strategy';
-    form_source?: 'ce_bot_dashboard_form' | 'ce_bot_builder_form';
-    form_name?: 'ce_bot_quick_strategy_form';
+        | 'edit_strategy'
+    form_source?: 'ce_bot_dashboard_form' | 'ce_bot_builder_form'
+    form_name?: 'ce_bot_quick_strategy_form'
     strategy_type?:
         | `d'alembert`
         | `martingale`
         | `oscar's-grind`
         | `reverse martingale`
         | `reverse d'alembert`
-        | `1-3-2-6`;
-    strategy_switcher_mode?: 'learn more' | 'trade parameters';
-    asset_type?: string;
-    trade_type?: string;
-    trade_type_mode?: string;
-    duration_type?: 'ticks' | 'seconds' | 'minutes' | 'hours' | 'days';
-    parameter_type?: 'initial-stake' | 'duration' | 'profit-threshold' | 'loss-threshold' | 'size-unit' | 'max-stake' | string;
-    parameter_field_type?: 'input' | 'dropdown' | 'slider' | 'checkbox' | 'number';
-    parameter_value?: string | number | boolean;
-    plus_minus_push?: 'yes' | 'no';
-    manual_parameter_input?: 'yes' | 'no';
-    dont_show_checkbox?: 'yes' | 'no';
-    cta_name?: 'edit_the_amount' | 'yes_continue';
-    learn_more_title?: string;
-};
+        | `1-3-2-6`
+    strategy_switcher_mode?: 'learn more' | 'trade parameters'
+    asset_type?: string
+    trade_type?: string
+    trade_type_mode?: string
+    duration_type?: 'ticks' | 'seconds' | 'minutes' | 'hours' | 'days'
+    parameter_type?:
+        | 'initial-stake'
+        | 'duration'
+        | 'profit-threshold'
+        | 'loss-threshold'
+        | 'size-unit'
+        | 'max-stake'
+        | string
+    parameter_field_type?: 'input' | 'dropdown' | 'slider' | 'checkbox' | 'number'
+    parameter_value?: string | number | boolean
+    plus_minus_push?: 'yes' | 'no'
+    manual_parameter_input?: 'yes' | 'no'
+    dont_show_checkbox?: 'yes' | 'no'
+    cta_name?: 'edit_the_amount' | 'yes_continue'
+    learn_more_title?: string
+}
```

Comment:
everything before this line was aligned automatically as a result of pre-commit hook execution.